docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -219,18 +235,22 @@ public class ReactiveGreetingResource {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     public String hello() {
-        return "hello";
+        return "Hello RESTEasy Reactive";
     }
 }
 ----
 
-It's a very simple REST endpoint, returning "hello" to requests on "/hello".
+It's a very simple REST endpoint, returning "Hello RESTEasy Reactive" to requests on "/hello".
+As it uses RESTEAsy Reactive, this method is called on the I/O thread.
+
+NOTE: to instruct Quarkus to invoke this method on a _worker_ thread, annotate it with the `io.smallrye.common.annotation.Blocking` annotation.
+You can use `@Blocking` on a method, class or enable it for the whole application by annotated an `Application` class.
```

Comment:
```suggestion
You can use `@Blocking` on a method, class or enable it for the whole application by annotated an `Application` class.
```
I think the notion of `Application` class is not sufficiently clear. Could you explain what that means? (or point to a guide where it's explained?)